public/javascripts/Gallery/src/modal/Modal.js

```diff
@@ -149,10 +175,10 @@ function Modal(uiModal) {
             sg.modal().pano.getPosition, getPanoId,
             function () { return properties['street_edge_id']; }, function () { return properties['region_id']; },
             sg.modal().pano.getPov, sg.cityName, false,
-            function() { sg.tracker.push('GSVInfoButton_Click', { panoId: getPanoId() }); },
-            function() { sg.tracker.push('GSVInfoCopyToClipboard_Click', { panoId: getPanoId() }); },
-            function() { sg.tracker.push('GSVInfoViewInGSV_Click', { panoId: getPanoId() }); },
-            function() { return properties['label_id']; }
+            function () { sg.tracker.push('GSVInfoButton_Click', { panoId: getPanoId() }); },
```

Comment:
Let's keep no space between these! I think that's what we have in the style guide?

Reply:
Sorry about this! My auto formatter changed it and I didn't notice.